pkgs/development/python-modules/m3u8/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, python, fetchFromGitHub, requests, iso8601, bottle, pytest, pytestcov }:
+
+buildPythonPackage rec {
+  pname = "m3u8";
+  version = "0.5.2";
+
+  src = fetchFromGitHub {
+    owner = "globocom";
+    repo = pname;
+    rev = version;
+    sha256 = "0p6wmwv1nfa5pyakq5d55w9v142z5ja3db3s3qr44kx895d9lhng";
+  };
+
+  checkInputs = [ bottle pytest pytestcov ];
+
+  checkPhase = ''
+    pytest tests/test_parser.py
+    pytest tests/test_model.py
+    pytest tests/test_variant_m3u8.py
```

you can replace those 3 lines, with below
```suggestion
     pytest tests/test_{parser,model,variant_m3u8}.py
```